alws/routers/errata.py

```diff
@@ -36,26 +38,37 @@
 
 @router.post("/new/", response_model=errata_schema.CreateErrataResponse)
 async def create_new_errata_record(
-    errata: errata_schema.BaseErrataRecord,
-    db: AsyncSession = Depends(AsyncSessionDependency(key=get_async_db_key())),
+    errata: errata_schema.BaseErrataRecord
 ):
-    record = await errata_crud.create_new_errata_record(
-        db,
-        errata,
-    )
-    return {"ok": bool(record)}
+    try:
+        errata_data = errata.model_dump()
+
+        # Convert date fields to strings
+        errata_data["issued_date"] = errata_data["issued_date"].isoformat()
+        errata_data["updated_date"] = errata_data["updated_date"].isoformat()
+
+        create_new_errata.send(errata)
```

Comment:
Ok, I don't really get this, or maybe I'm confused. You dump the model into `errata_data`, update the `_date` fields and do nothing with with later. If we want to update the `_date` fields, can't we just update the `errata` object?
And later, in https://github.com/AlmaLinux/albs-web-server/pull/1042/files#diff-15431caf390db7ef3490fc6e9a55950ee5edfe95773308b6c7d32b58f3c80f47R918, you wouldn't need to create an BaseErrataRecord.

Reply:
Yeah, I forgot to replace it here.
Without dumping this error happens `TypeError: Object of type BaseErrataRecord is not JSON serializable` when sending. And it requires to convert the date so that's why it looks like this

Reply:
oh right, dramatiq expects objects to be JSON serializable, thanks for the clarification